### ISSUE TITLE
Host checks does not work if we have 2 hybrid gateways

### ISSUE THREAD

**Issue by ilijabojanovic** [TykTechnologies/tyk#2036]
**Do you want to request a *feature* or report a *bug*?**
BUG
**What is the current behavior?**
When we have 2 slaved gateways on same redis, host check does not work on second gateway. If 1st gateway detect that some host is down, that info will be not propagated to second gateway.
**What is the expected behavior?**
If host under check is down, all gateway should exclude that host from routing list
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
1. Start 2 hybrid gateways
2. Create api definition with load balancing and enable host checks 

Gateway config for host checks:
```json
"uptime_tests": {
        "disable": false,
        "config": {
            "failure_trigger_sample_size": 2,
            "time_wait": 10,
            "checker_pool_size": 50,
            "enable_uptime_analytics": false
        }
```
**Which versions of Tyk affected by this issue? Did this work in previous versions of Tyk?**
Stable and Master

**Comment by ahmedkaballo** [comment]
hello 
we contacted the  support team and they provided us with test license for two week for testing to nodes architecture but they didn't provide us with a documentation of how to add another gateway.
so how can i add two gateways??
should i just configure the second gateway to reach the db and configure a load balancer with floating ip for the two gateways?

**Comment by ilijabojanovic** [comment]
Verified

**Comment by buger** [comment]
https://github.com/TykTechnologies/tyk/pull/2049 was merged to `%!s(MISSING)` branch
> Created via API

**Comment by buger** [comment]
https://github.com/TykTechnologies/tyk/pull/2049 was merged to `release-2.7` branch
> Created via API

**Comment by buger** [comment]
https://github.com/TykTechnologies/tyk/pull/2049 was merged to `release-2.8` branch
> Created via API

**Comment by buger** [comment]
<small>Test small text</small>

**Comment by buger** [comment]
https://github.com/TykTechnologies/tyk/pull/2049 was merged to `release-2.7` branch
<details><summary></summary>Created via API</details>

**Comment by buger** [comment]
https://github.com/TykTechnologies/tyk/pull/2049 was merged to `release-2.8` branch
<details><summary></summary>Created via API</details>
